Source/Gateway/Concepts/Logins/MimeType.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Dolittle.Concepts;
+
+namespace Concepts.Logins
+{
+    public class MimeType : ConceptAs<string>
```

Comment:
Same as for Logo Concept - we don't need this and it is not vital piece for the Domain. In fact, the information about its mime type is implicit through its existence on disk and its format (jpg, png... )